druntime/src/core/internal/atomic.d

```diff
@@ -1151,4 +1199,4 @@ string simpleFormat(string format, scope string[] args)
         break;
     }
     return result;
-}
+}
```

Comment:
newline at eof missing

Reply:
I edited this on the github mobile app, I have no idea how to change this.